website/blog/2021-02-08-64updates.md

```diff
@@ -0,0 +1,170 @@
+
+This feature is still considered experimental in 0.64, because it is not finished in React Native Core. Meaning we may need to change the ABI surface in future releases.
+
+## 🎯 Easy opt-in to Hermes
+
+Performance is a huge deal when developing in general, but especially so when it comes to using React Native. We're pretty invested is using Hermes and while there are teams using it in production, support is still experimental and we do not have the full Hermes debugger or Hermes in C# capabilities online yet. We would still love to know if you try Hermes and run into any issues, so go ahead and [give it a try today!]((https://microsoft.github.io/react-native-windows/docs/next/hermes))
+
+## 🎁 Get the goodness
+The best way to upgrade from 0.62 or 0.63 to this latest **0.64** release is to:
+- Create a brand new blank 0.64 using `react-native-windows-init`
+- Copy over the content of your older versions project
+
+Moving forward we are looking to provide a diff tool to help make this easier, this tool would operate similarly to `react-native-upgrade-helper`.
+
+To get the full list of release details, including breaking changes, check out our [release notes over on our repo.](https://github.com/microsoft/react-native-windows/releases/tag/react-native-windows_v0.64.0-preview.1)
+
+# 📚 Growing Windows community module support
+
+Over the last couple of months, the team has made a concerted effort to increase the number of modules that target React Native for Windows. We know that modules are an important part of React Native ecosystem, so we are excited to share that we’ve added Windows implementations for several modules, including:
+
+- [react-native-tts](https://github.com/ak1394/react-native-tts)
+- [react-native-codepush](https://github.com/microsoft/react-native-code-push)
+- [react-native-sketch-canvas](https://github.com/creambyemute/react-native-sketch-canvas)
+- [react-native-pdf](https://github.com/wonday/react-native-pdf)
+- [react-native-print](https://github.com/christopherdro/react-native-print)
+- [react-native-device-info](https://github.com/react-native-device-info/react-native-device-info)
+- [react-native-orientation-locker](https://github.com/wonday/react-native-orientation-locker)
+- [rn-fetch-blob](https://github.com/joltup/rn-fetch-blob) ([module unmaintained](https://github.com/joltup/rn-fetch-blob/issues/666), but [Windows code in PR](https://github.com/joltup/rn-fetch-blob/pull/701))
+- [react-native-splash-screen](https://github.com/crazycodeboy/react-native-splash-screen) ([in PR](https://github.com/crazycodeboy/react-native-splash-screen/pull/514))
+

```

Comment:
Should we also add the react-native-fs module which is also in PR? https://github.com/itinance/react-native-fs/pull/952